backend/frontend/urls.py

```diff
@@ -68,9 +68,14 @@
     ),
     # Course component details
     path(
-        "project/<int:project_id>",
+        "project_details/<int:id>/<content_type>",
```

Comment:
The suggest was for the urls to looks something like: `project_details/project` with the id at the end.

The way you are currently doing it is confusing because if the first part is `project_details` what's the use for `project` at the end?

Reply:
I made a typo on the above review, it should be `progress_details/project`